contracts/HydraDelegation/HydraDelegation.sol

```diff
@@ -87,6 +87,13 @@ contract HydraDelegation is
         _distributeDelegationRewards(staker, reward, epochId);
     }
 
+    /**
+     * @inheritdoc IHydraDelegation
+     */
+    function setDelegationPoolStaker(address staker) external onlyHydraChain {
```

Comment:
"initDelegationPool" sounds better to me

Reply:
Yeah, I actually thought about this, but we are actually initializing just the staker property. However, will rename it because I like it more, too.